compliance-web/src/components/App/ContinuationReports/ContinuationReportEntryModal.tsx

```diff
@@ -97,10 +97,25 @@ const ContinuationReportEntryModal: React.FC<ContinuationReportEntryModal> = ({
     useDeleteContinuationReportEntry(onSuccess);
 
   const onSubmitHandler = (data: ContinuationReportSchemaType) => {
+    let dateToUse;
+    
+    if (data.dateOfEntry?.isValid()) {
+      if (data.dateOfEntry.hour() === 0 && data.dateOfEntry.minute() === 0) {
```

Comment:
We should check for the form field dirtiness instead of values as 0, because there might be cases where they might insert 0, and would like to capture it like that.
we should check if the input fields are empty instead of 0.